samples/betaFeatures.js

```diff
@@ -0,0 +1,586 @@
+      const response = data[0];
+      const transcription = response.results
+        .map(
+          result =>
+            ` Channel Tag: ` +
+            result.channelTag +
+            ` ` +
+            result.alternatives[0].transcript
+        )
+        .join('\n');
+      console.log(`Transcription: \n${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeMultiChannel]
+}
+
+function speechTranscribeMultichannelGCS(gcsUri) {
+  // [START speechTranscribeMultichannelGcs]
+
+  //const fs = require('fs');
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    audioChannelCount: 2,
+    enableSeparateRecognitionperChannel: true,
+    enableAutomaticPunctuation: true,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(
+          result =>
+            ` Channel Tag: ` +
+            result.channelTag +
+            ` ` +
+            result.alternatives[0].transcript
+        )
+        .join('\n');
+      console.log(`Transcription: \n${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeMultichannelGcs]
+}
+
+// # MULTI LANGUAGE
+
+function speechTranscribeMultilang(speechFile) {
+  //[START speechTranscribeMultilang]
+
+  const fs = require('fs');
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    content: fs.readFileSync(speechFile).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speechTranscribeMultilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speechTranscribeMultilangGCS]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
```

Comment:
I'd prefer to omit this parameter (and use a mono audio file) if possible, since this sample isn't about multi-channel.

Reply:
This was fixed in last commit

Reply:
Agree with jerjou – I still see `audioChannelCount` in the sample here